packages/e2e-tests/tests/breakpoints-05.test.ts

```diff
@@ -33,6 +34,12 @@ test(`breakpoints-05: Test interaction of breakpoints with debugger statements`,
     lineNumber: 8,
     url,
   });
+
+  // This slight delay ensures that the breakpoint removal has had time to
+  // be processed and synched to the server, otherwise we could get racy behavior
+  // with the next rewind call.
+  await delay(1_000);
```

Comment:
If this is necessary, I think it should go into the `removeBreakpoint` helper rather than in a one-off test.

https://github.com/replayio/devtools/blob/798464e945a7dd0e0139ea5e6e067a84747ee9fa/packages/e2e-tests/helpers/source-panel.ts#L231-L255

Reply:
sounds good!